### PR TITLE
Adding lint rule for /inputpermission

### PR DESCRIPTION
**Pull request by sweetniamhy**
Adding a lint rule for using the Player.inputPermissions API

### CODE REVIEW

File: tools/eslint-plugin-minecraft-linting/src/Rules/AvoidUnnecessaryCommand.test.ts [hunk 1]
```diff
@@ -611,6 +611,21 @@ describe('AvoidUnnecessaryCommand', () => {
                         },
                     ],
                 },
+                {
+                    code: `const p = player.runCommandAsync('/inputpermission query @s movement enabled');`,
```
Comment (zaidraufskyboxlabs) [inline tools/eslint-plugin-minecraft-linting/src/Rules/AvoidUnnecessaryCommand.test.ts:615]: Hmmm I'm not super familiar with lint rules (so feel free to correct me if I'm wrong or wait for Jake/Raph) but would this only work for that specific string? I think we'd want something a bit more general like just `/inputpermission` so if it appears anywhere we can suggest replace with script method

Comment (zaidraufskyboxlabs) [reply tools/eslint-plugin-minecraft-linting/src/Rules/AvoidUnnecessaryCommand.test.ts:615]: It looks like the other ones don't do this though so I'm really not sure 😅

**Review by zaidraufskyboxlabs — APPROVED** [review]
LGTM!

**Comment by sweetniamhy** [comment]
@rlandav @JakeShirley wondering about the beachball changefile check, is there anything I need to do? The link still won't work as it'll be in 1.14.0 stable, but wanted to get things ready

**Comment by sweetniamhy** [comment]
> @rlandav @JakeShirley wondering about the beachball changefile check, is there anything I need to do? The link still won't work as it'll be in 1.14.0 stable, but wanted to get things ready

nvm found it in the contributing doc 😄

**Comment by JakeShirley** [comment]
@sweetniamhy Is this good to go?

**Comment by sweetniamhy** [comment]
> @sweetniamhy Is this good to go?

Oh, yes sorry I just need to run beachball on it I'll get that going 😄

**Comment by sweetniamhy** [comment]
@JakeShirley when you have a chance to review

**Review by JakeShirley — APPROVED** [review]

**Review by rlandav — APPROVED** [review]

**Pull request merged**